wg-data/README.md

```diff
@@ -0,0 +1,33 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/blob/master/generator/README.md
+--->
+# Data Working Group
+
+The WG "Data" is focused on enhancing the support for data/metadata-related tasks within Kubeflow, with a specific focus on the Spark operato and Model Registry. The group aims to simplify and improve data processing between various stages of ML lifecycle. For example, from Data Preparation to model training and fine-tuning. The group also aims to facilitate the ML model's metadata management, while ensuring seamless integration with other Kubeflow components. The goal of Spark on Kubernetes Operator is to simplify the capability of running Apache Spark on Kubernetes. It automates deployment and simplifies lifecycle management of Spark Jobs on Kubernetes. The goal of Model Registry is gather, analyze, and develop model registry requirements of Kubeflow community users.
+
+The [charter](charter.md) defines the scope and governance of the Data Working Group.
+
+## Meetings
+* KF Model Registry community meeting (US/EMEA): [Mondays at 7:00PM-8:00PM Europe/Madrid]() (biweely - every other Monday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=7:00PM-8:00PM&tz=Europe%2FMadrid).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1DmMhcae081SItH19gSqBpFtPfbkr9dFhSMCgs-JKzNo/edit?usp=sharing).
+
```

Comment:
```suggestion
* The Feast community meeting (US/EMEA): [Tuesdays at 8:00 PM Europe/Madrid]() (bi-weekly - every other Tuesday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=7:00PM-8:00PM&tz=Europe%2FMadrid).
* [Meeting notes and Agenda](https://docs.google.com/document/d/1MqNNhPCi1vvcYv71j3N0uBb4qIbl2ikaFO8tKohu12I/edit#heading=h.6js8vamr57ls).
```

Reply:
adding to yaml.

Reply:
done with fa3c3187d1d731bfe723c9c9b633a868baad051c in the yaml.